src/index.html

```diff
@@ -13,14 +13,17 @@ <h1 class="display-4"> <img class="welcome" src="{{site.baseurl}}/assets/images/
 <div class="container">
     <div class="row">
         <div class="col text-center">
-            <h2>Software</h2>
-            <a class="btn btn-primary btn-lg" href="./software/introduction/">Get Started</a>
-            <img class="welcome" src="{{site.baseurl}}/assets/images/geotab-drive-partner-add-ins-icon.svg" />
+            <h2>Get Started</h2>
```

Comment:
Can we keep 2 separate titles? Similar to:

```suggestion
            <div class="container">
    <div class="row">        
        <div class="col text-center">    
            <h2>Get Started with</h2>        
            <a class="btn btn-primary btn-lg" href="./software/introduction/">Software</a>
            <img class="welcome" src="{{site.baseurl}}/assets/images/geotab-drive-partner-add-ins-icon.svg" alt="" />
        </div>
        <div class="col text-center">  
            <h2>Get Started with</h2>             
            <a class="btn btn-primary btn-lg" href="./hardware/introduction/">Hardware</a>
            <img class="welcome" src="{{site.baseurl}}/assets/images/geotab-drive-partner-add-ons-icon.svg" alt=""/>
        </div>
    </div>
</div>
```